third-party/genuine-fantasy-press/forgotten-secrets/items.xml

```diff
@@ -0,0 +1,1324 @@
+		</setters>
+	</element>
+
+	<element name="Graveseekers" type="Magic Item" source="The Compendium of Forgotten Secrets" id="ID_GFP_COFSA_MAGIC_ITEM_WONDROUS_ITEM_GRAVESEEKERS">
+		<description>
+			<p>These eyeglasses shine with a faint blue sheen in the dark, concealing the eyes of the person wearing them. While wearing them, you can clearly see the exact location of any corpse or undead creature within 30 feet of you. </p>
+		</description>
+		<setters>
+			<set name="category">Wondrous Items</set>
+			<set name="cost" currency="gp">0</set>
+			<set name="weight" lb="0">0 lb.</set>
+			<set name="type">Wondrous Item</set>
+			<set name="rarity">Uncommon</set>
+			<set name="slot">head</set>
+		</setters>
+	</element>
+
+	<element name="Unlucky Dice" type="Magic Item" source="The Compendium of Forgotten Secrets" id="ID_GFP_COFSA_MAGIC_ITEM_WONDROUS_ITEM_UNLUCKY_DICE">
+		<description>
+			<p>A pair of six-sided dice that have skulls instead of regular pips. As an action, you can roll these dice. When you do, roll 2d6. If the total is odd, you take necrotic damage equal to the result. If the total is even, you gain temporary hit points equal to the result that last 1 hour. </p>
+		</description>
+		<setters>
+			<set name="category">Wondrous Items</set>
+			<set name="cost" currency="gp">0</set>
+			<set name="weight" lb="0">0 lb.</set>
+			<set name="type">Wondrous Item</set>
+			<set name="rarity">Uncommon</set>
+		</setters>
+	</element>
+
+	<element name="Adhesive Armor" type="Magic Item" source="The Compendium of Forgotten Secrets" id="ID_GFP_COFSA_MAGIC_ITEM_ARMOR_ADHESIVE_ARMOR">
+		<description>
+			<p>While wearing this green, ichorous armor, you gain advantage on any checks you make to initiate or maintain a grapple. Additionally, you can move along vertical surfaces on your turn without falling during your movement. You take no damage from falls of 30 feet or less.</p>
+		</description>
+		<setters>
+			<set name="category">Magic Armor</set>
+			<set name="cost" currency="gp">0</set>
+			<set name="type" addition="light, medium, or heavy">Armor</set>
+			<set name="rarity">Rare</set>
+			<set name="armor">ID_INTERNAL_ARMOR_GROUP_LIGHT|ID_INTERNAL_ARMOR_GROUP_MEDIUM|ID_INTERNAL_ARMOR_GROUP_HEAVY</set>
+			<set name="attunement">true</set>
+			<set name="slot">body</set>
+		</setters>
+	</element>
+
+	<element name="Jelly Helm" type="Magic Item" source="The Compendium of Forgotten Secrets" id="ID_GFP_COFSA_MAGIC_ITEM_WONDROUS_ITEM_JELLY_HELM">
+		<description>
+			<p>Vibrations in the air around you are amplified by the supple slime covering your head. You gain blindsight out to a range of 20 feet.</p>
+		</description>
+		<setters>
+			<set name="category">Wondrous Items</set>
+			<set name="cost" currency="gp">0</set>
+			<set name="weight" lb="0">0 lb.</set>
+			<set name="type">Wondrous Item</set>
+			<set name="rarity">rare</set>
+			<set name="attunement">true</set>
+			<set name="slot">head</set>
+		</setters>
+	</element>
```

Comment:
you can use `<grant type="Vision" id="ID_VISION_BLINDSIGHT" />` but no radius is available.